README.md

```diff
@@ -394,6 +394,79 @@ This creates a Python entrypoint, YAML configs and a Dockerfile. You can start f
 
 </details>
 
+#### MultimodalDocument
+  
+ A MultimodalDocument in Jina is a document composed by more than one chunks with different modalities.
```

Comment:
"composed by" should be "composed of"

"more than one chunks" should be "more than one chunk"

Reply:
multiple